keyboards/dnworks/frltkl/info.json

```
@@ -0,0 +1,113 @@
+{
+    "keyboard_name": "DN FRLTKL",
+    "maintainer": "itsme-zeix",
+    "manufacturer": "dnworks",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4C23",
+        "pid": "0x2935",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true    
+    },
+    "indicators": {
+        "caps_lock": "GP27",
+        "on_state": 1
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP4","GP3","GP2","GP1","GP29","GP28","GP19","GP20","GP10","GP11"],
+        "cols": ["GP26","GP25","GP24","GP23","GP22","GP21","GP6","GP5","GP0"]
```

human readable formatting
```suggestion
        "rows": ["GP4", "GP3", "GP2", "GP1", "GP29", "GP28", "GP19", "GP20", "GP10", "GP11"],
        "cols": ["GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP6", "GP5", "GP0"]
```